include/aikido/planner/World.hpp

```diff
@@ -7,9 +7,22 @@
 namespace aikido {
 namespace planner {
 
+/// A Kinematic world that contains a set of skeletons.
+/// It is expected that worlds do not share the same skeletons.
 class World
 {
 public:
+  // Encapsulates the state of the World.
+  struct State
+  {
+    std::vector<dart::dynamics::Skeleton::Configuration> configurations;
+
+    /// Returns true if two world states are the same.
```

Comment:
Is there any reason to use `equals` rather than `operator==`?

Reply:
I don't see any reason why we couldn't use `operator ==` here. If we define that, then we should also remember to define `operator !=`. 🙃 

Also, this likely should be flagged as `const`.